src/main/java/org/example/PhoneInterface.java

```diff
@@ -0,0 +1,17 @@
+package org.example;
+
+/**
+ * This is a interface. It promises 6 methods.
+ */
+public interface PhoneInterface {
```

Comment:
why do we need this interface? the Phone abstract class can do all the things the interface does and more